src/lib/components/Popover.svelte

```diff
@@ -11,6 +11,7 @@
   export let direction: "ltr" | "rtl" = "ltr";
   export let closeButton = false;
   export let invisibleBackdrop = false;
+  export let testId: string | undefined = undefined;
```

Comment:
How about defaulting to `"popover-component"`

Reply:
Yes, better! Changed, thank you for the suggestion